ansible.cfg

```diff
@@ -5,7 +5,7 @@ roles_path = ~/.ansible/roles/opensrp:./roles
 #
 # Use Mitogen connection backend by default for faster execution
 # See: https://mitogen.networkgenomics.com/ansible_detailed.html
-strategy_plugins = ~/.virtualenvs/opensrp/lib/python3.6/site-packages/ansible_mitogen/plugins/strategy
+strategy_plugins = ~/.virtualenvs/opensrp/lib/python3.6/site-packages/ansible_mitogen/plugins/strategy:.venv/lib/python3.6/site-packages/ansible_mitogen/plugins/strategy:.venv/lib/python3.7/site-packages/ansible_mitogen/plugins/strategy
```

Comment:
@gstuder-ona would you rather go with 
```
export ANSIBLE_STRATEGY_PLUGINS=.venv/lib/python3.7/site-packages/ansible_mitogen/plugins/strategy
```
The default path is something that matches what is in our admin host but not necessary in our dev machines. I use python3.8  environment for my local setups and the export is how the plugin in my environment is picked up.

Reply:
I'm not that worried if others disagree, but I'd argue that a local .venv should be a standard path option here in any case - it's less arbitrary than a particularly-named virtualenv?

The python3.6 path though should definitely go, agreed.

Happy to have you decide and edit/commit if you want to push the patch more quickly, or I can later.

Reply:
also, arguably, we could use a .pyenv file to better-enforce a particular python version - ansible tends to be finicky about that and I've definitely wasted time on version-related bugs.  

Out of scope for here, just sayin'

Reply:
Your additions highlighted it and in this case it seemed at odds with what I am used to. I will perhaps ask @jasonrogena to weigh in on this.

Reply:
Seems the list might end up being a long one. The reasoning behind pinning ".virtualenv" and Python 3.6 is because that is what is set up in Ona's bastion host. Since this, at least to me, appears trivial, we can merge and observe how well the fix ages.